api/query.go

```diff
@@ -217,6 +217,13 @@ func (e *Endpoint) SlicePost(ctx *gin.Context) {
 	e.slice(ctx, query)
 }
 
+// FencePost godoc
+// @Summary  Returns traces along an arbitrary path, such as a well-path
+// @Param    query  query  string  True  "Urlencoded/escaped SliceQuery"
+// @Accept   application/json
+// @Produce  application/octet-stream
```

Comment:
Right, I'll fix it! As Sveinung pointed out, we still need metadata! Since we have interpolation, you might not get back the exact traces you asked for, hence a list of actual cdp value is a must to return